charts/httpd/templates/httpdRestart.yaml

```
@@ -0,0 +1,64 @@
+{{- if .Values.httpdRestart.enable }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+rules:
+  - apiGroups: ["apps", "extensions"]
+    resources: ["deployments"]
+    resourceNames: [{{ include "httpd.fullname" . }}]
+    verbs: ["get", "patch", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "httpd.fullname" . }}-restart-deployment
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-cronjob
```

```suggestion
  name: {{ include "httpd.fullname" . }}-restart-deployment
```

- Homogeneity with the other objects created withing the same functional scope (svcaccount, role, etc.)
- putting the type of the object in the name is usually duplicating information